unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+project(unit-tests)
+
+find_package(CMakeRC REQUIRED)
```

Comment:
I think CMakeRC is not needed, since all MPD files already embedded in cpp

Reply:
Thanks, done